packit/config/package_config.py

```diff
@@ -275,4 +275,4 @@ def parse_loaded_config(
         return package_config
     except Exception as ex:
         logger.error(f"Cannot parse package config. {ex}.")
-        raise Exception(f"Cannot parse package config: {ex}.")
+        raise PackitConfigException(f"Cannot parse package config: {ex}.")
```

Comment:
What about using different exception (=creating a new subclass of `PackitConfigException`) to differentiate the "not found" and "cannot parse" scenario?

Reply:
We are just [a small step](https://github.com/dhodovsk/avocado/pull/1) from packit config not being required, so not found would get unused soon.

Reply:
That's not a problem -- you will ignore `PackitNotFoundConfigException` in the future (or it will never be raised) and catch the `PackitWrongConfigException`...

But do as you wish. It was only meant to avoid catching no-config scenario in the service code:

https://github.com/packit-service/packit-service/blob/081fab76b46b01a6ca6a80ce1ed13e10975ff7f8/packit_service/worker/github_handlers.py#L85-L98

Reply:
I added incorect title to the PR. This is about invalid, becase missing would have to be raised here https://github.com/packit-service/packit/blob/1f22e358760394b54c22f24bbefa6594e42db003/packit/config/package_config.py#L255

Is that what you meant?

Reply:
I mean different exceptions = different type of exceptions...

(So we can differentiate this directly with multiple `except` block instead of parsing exception message.)

Reply:
I am a bit confused. There is no exception raised when the packit config is not found in the and right now there is no exception message parsing in https://github.com/packit-service/packit-service/pull/328
Would you like to change that behaviour?

Reply:
Yes, I see that there is only a warning... OK, we can leave it as is. (Or raise the specific exception only in the service code.)